apps/finance/contracts/Finance.sol

```diff
@@ -211,6 +211,34 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
         );
     }
 
+    /**
+    * @notice Create a new payment of `@tokenAmount(_token, _amount)` to `_receiver` for '`_reference`'
+    * @dev Note that this function is protected by the `CREATE_PAYMENTS_ROLE` but uses `MAX_UINT64`
+    *      as its interval auth parameter. While this protects against most cases (you typically want
+    *      to set a baseline requirement for the interval time), it does mean users can't grant a
+    *      permission that has a upperbound requirement for the interval time.
+    * @param _token Address of token for payment
+    * @param _receiver Address that will receive payment
+    * @param _amount Tokens that are paid every time the payment is due
+    * @param _reference String detailing payment reason
+    */
+    function newImmediatePayment(address _token, address _receiver, uint256 _amount, string _reference)
+        external
+        authP(CREATE_PAYMENTS_ROLE, _arr(_token, _receiver, _amount, uint256(getTimestamp64()), uint256(MAX_UINT64), uint256(1)))
```

Comment:
We can use [`getTimestamp`](https://github.com/aragon/aragonOS/blob/dev/contracts/common/TimeHelpers.sol#L36) so we don't have to cast, right?

Reply:
Fixed in https://github.com/aragon/aragon-apps/pull/711.

Reply:
Cool, just one little comment, I see this one builds on top of #711 , so let's make sure it's not overwritten when merging.